pkg/services/ngalert/api/api_ruler_history.go

```diff
@@ -21,16 +22,27 @@ type HistorySrv struct {
 	hist   Historian
 }
 
+const labelQueryPrefix = "labels_"
```

Comment:
Chose to prefix all labels queries with a known prefix. This is so that:
- Existing query params ("from", etc) don't conflict with labels
- Future query params we might want to add don't break users' existing label queries (i.e. if someone queries for a label `folder`, and we add a query parameter called `folder` in the future, it won't override their label query)